src/lib/OpenEXRCore/chunk.c

```diff
@@ -70,6 +70,17 @@ extract_chunk_table (
             return ctxt->report_error (
                 ctxt, EXR_ERR_INVALID_ARGUMENT, "Invalid file with no chunks");
 
+        if (chunkbytes + chunkoff > (uint64_t) ctxt->file_size)
```

Comment:
fyi, can't remember but I do allow a stream implementation to return -1 if they do not know the file size, which should then return false and succeed for this, but just thought I'd mention it so you know - I have been a bit pedantic about checking that elsewhere, although the math here is pretty simple...